docs/refined/index.md

```diff
@@ -26,10 +26,10 @@ object RefinedReadConfig extends App {
 
   def prodConfig =
     (
-      nonEmpty(string("LDAP")) |@|
-        greaterEqual[W.`1024`.T](int("PORT")) |@|
-        nonEmpty(string("DB_URL")).optional |@|
-        size[Greater[W.`2`.T]](list(long("LONGVALS")))
+      nonEmpty(string.at("LDAP")) |@|
```

Comment:
Guess, I think we need some poofs now.
`nonEmtpy(string).at("LDAP") == nonEmpty(string.at("LDAP"))`

Reply:
done